.codeclimate.yml

```diff
@@ -3,6 +3,8 @@ engines:
     enabled: false
   eslint:
     enabled: true
+    exclude_paths:
+    - "test/**"
```

Comment:
This change is in response to discussion about analyze test directories; might be premature. 

Happy to move back to top level excludes if we preferred that?